src/test/java/com/google/api/codegen/testdata/discoveries/py/py_logging.v2beta1.json.baseline

```diff
@@ -208,30 +213,27 @@ BEFORE RUNNING:
 2. This sample uses Application Default Credentials for authentication.
    If not already done, install the gcloud CLI from
    https://cloud.google.com/sdk/ and run
-   'gcloud beta auth application-default login'
+   `gcloud beta auth application-default login`
 3. Install the Python client library for Google APIs by running
-   'pip install --upgrade google-api-python-client'
+   `pip install --upgrade google-api-python-client`
 """
 
 from googleapiclient import discovery
 from oauth2client.client import GoogleCredentials
 
-# Authentication is provided by the 'gcloud' tool when running locally
+# Authentication is provided by the `gcloud` tool when running locally
 # and by built-in service accounts when running on GAE, GCE, or GKE.
-# See https://developers.google.com/identity/protocols/application-default-credentials for more information.
+# For more information, see
+# https://developers.google.com/identity/protocols/application-default-credentials
 credentials = GoogleCredentials.get_application_default()
 
 # Construct the logging service object (version v2beta1) for interacting
 # with the API. You can browse other available API services and versions at
 # https://developers.google.com/api-client-library/python/apis/
 service = discovery.build('logging', 'v2beta1', credentials=credentials)
 
-
-# TODO: Change placeholders below to appropriate parameter values for the 'delete' method:
-
-# * The resource name of the metric to delete. Example:
-#   `"projects/my-project-id/metrics/my-metric-id"`.
-metricName = 'projects/{MY-PROJECT}/metrics/{MY-METRIC}'
+# The resource name of the metric to delete. Example: `"projects/my-project-id/metrics/my-metric-id"`.
+metricName = 'projects/{MY-PROJECT}/metrics/{MY-METRIC}'  # TODO: Update placeholder value.
```

Comment:
I changed it so this comment is printed after each value to replace, I think `value` makes more sense given it's referencing a single field?